qt_dotgraph/src/qt_dotgraph/dot_shapes.py

```diff
@@ -19,27 +19,28 @@ def paint(self, painter, option, widget):
                            self._bounding_box.height() - self._bounding_box.height() * 0.1)
         painter.drawRect(rectangle)
         # Top line
-        painter.drawLine(rectangle.topLeft().x() + self._bounding_box.height() * 0.1,
-                         self._bounding_box.topLeft().y(),
-                         self._bounding_box.topRight().x(),
-                         self._bounding_box.topRight().y())
+        painter.drawLine((int)(rectangle.topLeft().x() + self._bounding_box.height() * 0.1),
+                         (int)(self._bounding_box.topLeft().y()),
+                         (int)(self._bounding_box.topRight().x()),
+                         (int)(self._bounding_box.topRight().y()))
```

Comment:
`int` doesn't need to go between brackets.
This is not a c-style cast, it's constructing an object of type `int` from a float.

```suggestion
        painter.drawLine(int(rectangle.topLeft().x() + self._bounding_box.height() * 0.1),
                         int(self._bounding_box.topLeft().y()),
                         int(self._bounding_box.topRight().x()),
                         int(self._bounding_box.topRight().y()))
```

Reply:
same everywhere else

Reply:
Ah, makes sense!